b2b-sample/guardio-insurance-sample-application/components/settingsComponents/identity-providers/identity-providers.js

```diff
@@ -75,98 +74,39 @@ export default function IdentityProviders() {
         setSelectedTemplate(undefined)
     };
 
-    const onIdPSave = async (formValues, template) => {
-
-        const FIRST_ENTRY = 0;
-        let model = { ...template.idp };
+    const onIdpCreated = (response) => {
+        if (response) {
+            successTypeDialog(toaster, "Success", "Identity Provider Created Successfully");
```

Comment:
`Identity Provider is created successfully

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/130d957728499e5e0e731eba0b663086f931fe95